src/iter/repeat.rs

```diff
@@ -111,10 +111,19 @@ pub struct RepeatN<T: Clone + Send> {
 ///
 /// ```
 /// use rayon::prelude::*;
-/// use rayon::iter::repeatn;
-/// let x: Vec<(i32, i32)> = repeatn(22, 3).zip(0..3).collect();
+/// use rayon::iter::repeat_n;
+/// let x: Vec<(i32, i32)> = repeat_n(22, 3).zip(0..3).collect();
 /// assert_eq!(x, vec![(22, 0), (22, 1), (22, 2)]);
 /// ```
+pub fn repeat_n<T: Clone + Send>(elt: T, n: usize) -> RepeatN<T> {
+    RepeatN {
+        element: elt,
+        count: n,
+    }
+}
+
+#[doc(hidden)]
```

Comment:
I don't think we should hide it from docs.
```suggestion
```

(We hide `find` and `position` because they only exist anymore to guide people porting from `Iterator`, versus something we changed ourselves.)

Reply:
Thanks! I have removed `#[doc(hidden)]`.